lib/src/main/java/com/hootsuite/nachos/NachoTextView.java

```diff
@@ -148,6 +149,7 @@ public void onChipClick(Chip chip, MotionEvent motionEvent) {
     // Touch events
     @Nullable
     private OnChipClickListener mOnChipClickListener;
+    private GestureDetector gestureDetector;
```

Comment:
@sherry-yuan-hs gestureDetector is needed in onTouchEvent() but we can't create it there. The detector needs to receive all events in order to tell if the up action is a click or not.